models/error.go

```diff
@@ -1385,3 +1385,46 @@ func (err ErrNotValidReviewRequest) Error() string {
 		err.UserID,
 		err.RepoID)
 }
+
+//
+// ___________                  .__             __________        _____
+// \_   _____/__________   ____ |__| ____   ____\______   \ _____/ ____\___________   ____   ____   ____  ____
+//  |    __)/  _ \_  __ \_/ __ \|  |/ ___\ /    \|       _// __ \   __\/ __ \_  __ \_/ __ \ /    \_/ ___\/ __ \
+//  |     \(  <_> )  | \/\  ___/|  / /_/  >   |  \    |   \  ___/|  | \  ___/|  | \/\  ___/|   |  \  \__\  ___/
+//  \___  / \____/|__|    \___  >__\___  /|___|  /____|_  /\___  >__|  \___  >__|    \___  >___|  /\___  >___  >
+//      \/                    \/  /_____/      \/       \/     \/          \/            \/     \/     \/    \/
+//
```

Comment:
I have a feeling that we should put the code to a correct place, instead of using this big banner 😂